app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> None:
+    store = {}
+
+    def inner(*args) -> str:
```

Comment:
Last remark: add an annotation for args. And I doubt about the return annotations. For outer function it should be  `Callable`, but for inner it should definitely not be `str`, I think `Any` fits better. Everything else looks good